client/stylesheets/_responsive.scss

```diff
@@ -298,6 +293,20 @@
 }
 
 @media (max-width: 650px) {
+    .profileCardInfo li:first-child {
+        font-size: 1rem !important;
+    }
+
+    .profileCardItem {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `profileCardItem` should be written in lowercase with hyphens